cloud/db_cloud_impl.cc

```diff
@@ -128,10 +129,26 @@ Status DBCloud::Open(const Options& opt, const std::string& local_dbname,
     if (st.ok()) {
       st = cenv->LoadCloudManifest(local_dbname, read_only);
     }
+    if (st.IsNotFound()) {
+      Log(InfoLogLevel::INFO_LEVEL, options.info_log,
+          "CLOUDMANIFEST not found in the cloud, assuming this is a new "
+          "database");
+      new_db = true;
+      st = Status::OK();
```

Comment:
Is this just convention to reset our status here? I don't think we check `st` after.

Reply:
This just means that we are ignoring the `IsNotFound` error. Yes, currently we don't check the status below, but it's a convention to keep status as `Status::OK`, unless there's an error.